### ISSUE TITLE
Pointer Events as a Replacement for Mouse, Touch, and Stylus Events

### ISSUE THREAD

**Issue by danwilson** [jsfest/sf-cfp#31]
This talk will introduce today's implementations of the new W3C spec for Pointer Events - a new standard to handle all pointer based input (mouse, touch, stylus, etc.) in one collection of events instead of specific ones for each input type.  Touchscreens are not always used with touch, and desktops are not always limited to a mouse and keyboard.

Pointer Events are already in heavy use with Windows 8+ and IE 10+.  I will discuss the basics of how Pointer Events work, as well as how they compare to how the `mousedown`, `touchstart`, and other events of which we are already familiar.

As an example, I will walk through a few gestures/interactions that Windows 8 apps and sites employ.  They will show how the use of a single event to support the multiple input types make our development easier.  And hopefully show that it's worth never assuming one input type per screen or device.

**Comment by mikeal** [comment]
This sounds like a great talk for JSConf but I don't know if it's a fit for JSFest.

> JSFest is about the present more than the future. Talks should be about new technology that you can already take for granted. JSFest is about sharing what early adopters and community leaders are using with a broader audience.

This stuff is really cool and, personally, I want to see this talk but this is hardly something we can take for granted yet :(

**Comment by danwilson** [comment]
That's true - it's not present across the browsers.  The only place it can be taken for granted currently is if you are developing native Windows 8 apps using HTML/JS and WinJS.  Thanks!
